routes/web.php

```diff
@@ -13,6 +20,44 @@
 |
 */
 
+Route::group([
```

Comment:
Only admin can access to the Admin Portal, maybe related to Authorization @hoaingodong1